specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json

```diff
@@ -5603,47 +5654,55 @@
           }
         },
         "x-ms-examples": {
-          "OAuthTokens_Delete": {
-            "$ref": "./examples/OAuthTokens_Delete.json"
+          "OAuthTokens_GetCascadeDeleteJobDetails": {
+            "$ref": "./examples/OAuthTokens_GetCascadeDeleteJobDetails.json"
           }
         }
-      }
-    },
-    "/oauth/tokens/:connect": {
-      "post": {
+      },
+      "put": {
         "tags": [
           "OAuthTokens"
         ],
-        "description": "Returns Connection link needed in the OAuth flow.",
-        "operationId": "OAuthTokens_GetOAuthConnectionLink",
-        "consumes": [
-          "application/json"
-        ],
+        "description": "Create remove job for OAuth token.",
```

Comment:
Maybe something more descriptive like: `Create a cascade delete job for OAuth tokens.`

Reply:
Updated.